src/lucky_record/queryable.cr

```diff
@@ -29,7 +29,7 @@ module LuckyRecord::Queryable(T)
   end
 
   def find(id)
-    id(id).limit(1).first
+    id(id).limit(1).first? || raise RecordNotFound.new
```

Comment:
Maybe you can pass in just the table name and the id: `RecordNotFound.new(id: id, table_name: @@table_name)`. That's probably all the error needs for now